# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/mew/fix-cite-config-bug.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This fixes a logic bug to make the cite behaviour configurable.

  - Rene Rebe <rene@exactcode.de>

--- mew-3.3/mew-draft.el	2003-06-19 05:58:13.000000000 +0000
+++ mew-3.3-fixed/mew-draft.el	2003-09-11 17:49:10.000000000 +0000
@@ -499,8 +499,8 @@
 	  (let ((mark-active t) (transient-mark-mode nil))
 	    (cond
 	     ;; arg will be effect in mew-cite-original
-	     ((and (mew-mark) (or (not mew-cite-ignore-mouse-region)
-				  (not (mew-mouse-region-p))))
+	     ((and (mew-mark) (and (not mew-cite-ignore-mouse-region)
+				   (not (mew-mouse-region-p))))
 	      (setq beg (region-beginning) end (region-end)))
 	     ((mew-header-p)
 	      ;; header exists in Message mode. Skip the header
